tests/src/test/java/ApplicationTests.java

```diff
@@ -136,6 +136,40 @@ void spawnWaves(){
         assertFalse(Groups.unit.isEmpty(), "No enemies spawned.");
     }
 
+    @Test
+    void checkInitialItems() {
```

Comment:
I don't really see the purpose of this test. It may not always be the case that every map has 200 copper as a starting item, and some maps may not have any starting items at all. If I ever increase the starting copper amount to *300* copper, this test will break.